mw-webapp/src/component/editableTextarea/editableTextarea.tsx

```diff
@@ -71,18 +74,22 @@ export const EditableTextarea = (props: EditableTextareaProps) => {
     }
   };
 
+  const handleChange = (event: ChangeEvent<HTMLTextAreaElement>) => {
+    const newText = event.target.value;
+    setText(newText);
+
+    if (event.target.parentNode instanceof HTMLElement) {
+      event.target.parentNode.dataset.replicatedValue = newText;
+    }
```

Comment:
Why do we need this lines?

Reply:
This is a mechanism for synchronizing data with the parent element when the value in the text field changes.

Reply:
Settings data in some dom element directly does not look like good react approach. Let'stry to get rid of it or provide more information